### ISSUE TITLE
A11y Issue: Headers on new pages don't receive focus

### ISSUE THREAD

**Issue by carltonsmith** [raft-tech/TANF-app#406]
# Bug 2

## Description 
- Focus gets stuck on where a button was on the previous screen

## screenshot(s) of  expected and current behavior, 
![](https://i.imgur.com/KrB7glf.png)

## steps to reproduce 
- Chrome or Safari MacOS
- Turn on Voiceover
- Go to authenticated TDP, fill out form, navigate to Submit with keyboard nav
- Activate Button

## severity
- Current state fails to meet WCAG/508 standards as focus getting stuck seems to prevent any status/state/success message from being read. This might be made less severe when distinct page titles are implemented, but more testing will be needed

## notes
- Robert noted that this should be as simple as the fix for the timeout modal's header. (tab-index -1)

**Comment by lfrohlich** [comment]
Merged 2/3/21
